ruma-events-macros/src/event.rs

```diff
@@ -244,7 +244,9 @@ fn expand_deserialize_event(
 
                         while let Some(key) = map.next_key()? {
                             match key {
-                                Field::Unknown => continue,
+                                Field::Unknown => {
+                                    let _: Box<::serde_json::value::RawValue> = map.next_value()?;
```

Comment:
There is [IgnoredAny](https://docs.rs/serde/1.0.112/serde/de/struct.IgnoredAny.html) for this use case.